cocotb/share/makefiles/Makefile.sim

```diff
@@ -89,15 +97,16 @@ SIM ?= icarus
 SIM_LOWERCASE := $(shell echo $(SIM) | tr A-Z a-z)
 
 # Directory containing the cocotb Makfiles (realpath for Windows compatibility)
-COCOTB_MAKEFILES_DIR := $(realpath $(shell cocotb-config --makefiles))
+COCOTB_MAKEFILES_DIR := $(realpath $(shell $(COCOTB_CONFIG_CMD) --makefiles))
 
 include $(COCOTB_MAKEFILES_DIR)/Makefile.deprecations
 
 HAVE_SIMULATOR = $(shell if [ -f $(COCOTB_MAKEFILES_DIR)/simulators/Makefile.$(SIM_LOWERCASE) ]; then echo 1; else echo 0; fi;)
 AVAILABLE_SIMULATORS = $(patsubst .%,%,$(suffix $(wildcard $(COCOTB_MAKEFILES_DIR)/simulators/Makefile.*)))
 
 ifeq ($(HAVE_SIMULATOR),0)
-    $(error "Couldn't find makefile for simulator: "$(SIM_LOWERCASE)"! Available simulators: $(AVAILABLE_SIMULATORS)")
+    $(error Couldn't find makefile for simulator: "$(SIM_LOWERCASE)"! \
+            Available simulators: $(AVAILABLE_SIMULATORS))
```

Comment:
Does this print the indentation?

Reply:
no

Reply:
Ah, I'd assumed this would also print over multiple lines, which it does not